etc/fdns.service

```diff
@@ -21,5 +21,98 @@ Restart=on-failure
 #    and change the service type from `forking' to `simple'.
 #StandardOutput=append:/tmp/fdns-log.txt
 
+AmbientCapabilities=
+LockPersonality=true
+RestrictNamespaces=ipc mnt pid uts
+RestrictRealtime=true
+RestrictSUIDSGID=true
+ProtectHostname=true
+PrivateMounts=true
+MountFlags=private
+ProtectKernelTunables=true
+ProtectKernelModules=true
+ProtectControlGroups=true
+
+# FJ: --read-only=/ (except: /dev, /proc /sys)
+ProtectSystem=strict
+
+# FJ: --private --private-tmp
+ProtectHome=true
+PrivateTmp=true
+
+# FJ: --private-dev --no…
+PrivateDevices=true
+DevicePolicy=closed
+
+# FJ: --caps.keep=kill,net-bind,setgid,setuid,sys-admin,sys-chroot
+CapabilityBoundingSet=CAP_KILL CAP_NET_BIND_SERVICE CAP_SETGID CAP_SETUID CAP_SYS_ADMIN CAP_SYS_CHROOT
+
+# FJ: --memor-deny-write-execute --nonewprivs --protocol=unix,inet,inet6
+MemoryDenyWriteExecute=true
+NoNewPrivileges=true
+RestrictAddressFamilies=AF_UNIX AF_INET AF_INET6
+
+# FJ: --seccomp.keep=@system-service,@mount,seccomp --seccomp.drop=@debug,@obsolete --seccomp.block-secondary
+SystemCallFilter=@system-service @mount seccomp
```

Comment:
SystemCallFilter is not supported on Ubuntu 16.04 LTS, which has systemd [229-4ubuntu21.23]